snyk-import/main.ts

```diff
@@ -0,0 +1,100 @@
+// deno-lint-ignore-file no-explicit-any
```

Comment:
Do we want to ignore this lint? It seems to suggest that's poor style.

Reply:
It can be poor style.. But this code makes use of `fetch()` APIs, which is probably TypeScript's greatest weakness... I could define the types returned by these REST APIs and use type assertions, but that is more code and it doesn't really provide any benefit. If my types are wrong or the API changes, the code still type checks, but it fails at runtime.

My compromise here is to simply use `any` to opt out of TypeScript type checks in a few places.

The majority of this code is still type checked. And **implicit** use of `any` is still disallowed, so the user can see where the "dangerous" code is.

It's also worth noting that we do not currently run `deno lint` in CI for this repository and the Phylum CLI does not do any type checking when it runs extensions. So this line isn't strictly necessary... But it makes it easier if/when we add real type checking in the future.